save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -89,18 +90,33 @@ class VulnerabilityController(
         description = "Update vulnerability.",
     )
     @ApiResponse(responseCode = "200", description = "Successfully updated vulnerability")
+    @RequiresAuthorizationSourceHeader
     @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
     fun update(
         @RequestBody vulnerabilityDto: VulnerabilityDto,
-        authentication: Authentication,
     ): Mono<StringResponse> = blockingToMono {
-        if (authentication.hasRole(Role.SUPER_ADMIN)) {
-            vulnerabilityService.update(vulnerabilityDto)
-        }
+        vulnerabilityService.update(vulnerabilityDto)
     }.map {
         ResponseEntity.ok("Vulnerability was successfully updated")
     }
 
+    @DeleteMapping("/delete")
+    @Operation(
+        method = "Delete",
+        summary = "Deleted vulnerability.",
+        description = "Deleted vulnerability.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully deleted vulnerability")
+    @RequiresAuthorizationSourceHeader
+    @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
+    fun delete(
+        @RequestParam name: String,
+    ): Mono<StringResponse> = blockingToMono {
+        vulnerabilityService.delete(name)
```

Comment:
suggestion to throw not found when there is an attempt to delete by invalid name

Reply:
Added